moveit_planners/pilz_industrial_motion_planner_testutils/include/pilz_industrial_motion_planner_testutils/center.h

```diff
@@ -57,4 +56,4 @@ std::string Center<ConfigType, BuilderType>::getConstraintName() const
 }
 }  // namespace pilz_industrial_motion_planner_testutils
 
-#endif  // CENTERAUXILIARY_H
+//#endif  // CENTERAUXILIARY_H
```

Comment:
and here... there are many more of these.  Please just remove these lines instead of commenting them out.

Reply:
Oh, I am so sorry, I searched with grep if any files have commented #endif but I just realized that I messed up the spelling of endif while searching and it returned nothing so I thought I removed all already. I just did that, I am so sorry I keep making extremely dumb mistakes oof.